src/aind_behavior_experiment_launcher/behavior_launcher/slims_picker.py

```diff
@@ -0,0 +1,270 @@
+                    subject = None
+            self.launcher.subject = subject
+
+        try:
+            self._slims_session = self.slims_client.fetch_models(
+                models.behavior_session.SlimsBehaviorSession, mouse_pk=self._slims_mouse.pk
+            )[-1]
+        except IndexError:  # empty list returned from slims
+            raise ValueError(f"No session found on slims for mouse {subject}.")
+
+        return self.launcher.session_schema_model(
+            experiment="",  # Will be set later
+            root_path=str(self.launcher.data_dir.resolve())
+            if not self.launcher.group_by_subject_log
+            else str(self.launcher.data_dir.resolve() / subject),
+            subject=subject,
+            notes=self._slims_session.notes,
+            experimenter=username if username is not None else [],
+            commit_hash=self.launcher.repository.head.commit.hexsha,
+            allow_dirty_repo=self.launcher.is_debug_mode or self.launcher.allow_dirty,
+            skip_hardware_validation=self.launcher.skip_hardware_validation,
+            experiment_version="",  # Will be set later
+        )
+
+    def pick_task_logic(self) -> _T:
+        """
+        Returns task_logic found as an attachment from session loaded from slims.
+
+        Returns:
+            TTaskLogic: Task logic found as an attachment from session loaded from slims.
+
+        Raises:
+            ValueError: If no valid task logic attachment is found.
+        """
+
+        try:  # If the task logic is already set (e.g. from CLI), skip the prompt
+            task_logic = self.launcher.task_logic_schema
+            assert task_logic is not None
+            return task_logic
+        except ValueError:
+            task_logic = None
+
+        if self._slims_session is not None:
+            # check attachments from loaded session
+            attachments = self.slims_client.fetch_attachments(self._slims_session)
+            try:  # get most recently added task_logic
+                response = [
+                    self.slims_client.fetch_attachment_content(attach).json()
+                    for attach in attachments
+                    if attach.name == "task_logic"
+                ][0]  # TODO: hardcoded attachment name here. Not sure where/how we should store this value
+            except IndexError:  # empty attachment list with loaded session
+                raise ValueError(
+                    "No task_logic model found on with loaded slims session for mouse"
+                    f" {self.launcher.subject}. Please add before continuing."
+                )
+
+            return self.launcher.task_logic_schema_model(**response)
+
+        else:
+            logger.info("No Slims session loaded.")
+
+    @override
+    def initialize(self) -> None:
+        """
+        Initializes the picker
+        """
+
+    def prompt_username(self, strict: bool = True) -> Optional[List[str]]:
```

Comment:
`prompt_experimenter` to align with the other pickers

Reply:
We are not yet sure if this validation will be necessary. For now, lets just default to use the same as DefaultBehaviorPicker.